src/runtime/gc_conservative.go

```diff
@@ -45,6 +45,7 @@ const (
 	bytesPerBlock      = wordsPerBlock * unsafe.Sizeof(heapStart)
 	stateBits          = 2 // how many bits a block state takes (see blockState type)
 	blocksPerStateByte = 8 / stateBits
+	markStackSize      = 4 * unsafe.Sizeof((*int)(nil)) // number of to-be-marked blocks to queue before forcing a rescan
```

Comment:
This number seems arbitrary but I guess it's reasonable.

Reply:
I basically had decided that it would be reasonable to spend 64 bytes on ARM, and then decided to reduce that to 16 on AVR. I basically just went with `4*(wordWidth^2)` bytes spent on the stack, since 256 bytes also seemed fine for AMD64 where there is plenty of memory.